package.json

```diff
@@ -86,13 +88,14 @@
     "@babel/core": "^7.19.0",
     "@babel/preset-env": "^7.19.0",
     "babel-loader": "^8.2.5",
-    "babel-plugin-prismjs": "^2.1.0",
+    "babel-plugin-prismjs": "github:queengooborg/babel-plugin-prismjs#patch-1-build",
```

Comment:
What's the rationale/reason for this patch?

Reply:
This is a built version of https://github.com/mAAdhaTTah/babel-plugin-prismjs/pull/331, which fixes https://github.com/mAAdhaTTah/babel-plugin-prismjs/issues/330.  Basically, the plugin does not specify the file extensions for imports, however ESM requires that file extensions are always specified.  Hopefully the PR will be merged soon so that we don't have to continue using my fork!